### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Nilanshrajput**
## Description
Improving Readme

issue #103 
## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/)
 
+## SyferText
+
+SyferText is a library for privacy preserving Natural Language Processing in Python. It leverage PySyft for doing Encrypted Computation (
```
Comment (kyscg) [inline README.md:9]:
```suggestion
SyferText is a library for privacy preserving Natural Language Processing in Python. It leverages PySyft for doing Encrypted Computation
```

**Comment by Nilanshrajput** [comment]
@cereallarceny can you look at this https://github.com/sourcerer-io/hall-of-fame for contribution section, adding all the contributors will add clutter and for libraries like Pysyft with a huge amount of contributors it won't be practical to add all.

**Comment by AlanAboudib** [comment]
Good work @Nilanshrajput I will add some modification to the SyferText intro section

**Comment by cereallarceny** [comment]
Personally @Nilanshrajput @AlanAboudib I'd prefer to go with All Contributors since we're doing that on all our other repos. Let's stick to a convention please. I don't think there's a need for repos like PySyft to go back and list every contributor. We can list primary team members and new contributors. Likewise, people can always make a PR to add themselves to the list, but I don't think it's reasonable to go and add a bunch of people retroactively from 3 years ago.

In the desire to stick to the format, let's use All Contributors. If you'd like, for repos with a lot of contributors, you can have it all done in a separate readme file (`CONTRIBUTORS.md`) that is linked to from the Contributors section on the main readme. How is that for a compromise? :)

**Comment by Nilanshrajput** [comment]
@cereallarceny sounds good, I will create a separate file for contributors!

**Comment by Nilanshrajput** [comment]
@all-contributors

**Comment by Nilanshrajput** [comment]
@all-contributors please add @AlanAboudib  for ideas, code, design, doc

**Comment by Nilanshrajput** [comment]
@all-contributors please add @Nilanshrajput  for ideas, code, design, doc
@all-contributors please add @bicycleman15 for ideas, code, design, doc

**Comment by Nilanshrajput** [comment]
@all-contributors please add @Nilanshrajput for ideas, code, design, doc

**Comment by Nilanshrajput** [comment]
@all-contributors please add @bicycleman15 for ideas, code, design, doc

**Comment by Nilanshrajput** [comment]
@all-contributors please add @dzlab  for ideas, code, design, doc

**Comment by Nilanshrajput** [comment]
@all-contributors please add @MarcioPorto  for ideas, code, design, doc

**Comment by Nilanshrajput** [comment]
@all-contributors please add @Nilanshrajput  for  code

**Comment by Nilanshrajput** [comment]
@all-contributors please add @bicycleman15 for code

**Comment by AlanAboudib** [comment]
@Nilanshrajput  In the introduction part of the README, can you please mention the usage cases instead (That I mentioned in my AMA https://www.youtube.com/watch?v=gtwOPxxJ5F4&t=2614s) So the two usage cases, and the two types of users.

You can also include the link to the AMA, as it includes an introduction to what SyferText.

**Comment by Nilanshrajput** [comment]
Okay, I was also thinking of it while watching your AMA  but forgot about it.

**Comment by AlanAboudib** [comment]
Great, I reminded you then ;)

**Comment by AlanAboudib** [comment]
Is this ready for review @Nilanshrajput , also, we will merge it into syfertext_0.1.0 branch

**Comment by Nilanshrajput** [comment]
@AlanAboudib  yup it's ready for review.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,22 @@
...
 [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/)
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
+
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+## SyferText
+
+SyferText is a library for privacy preserving Natural Language Processing in Python. It leverage PySyft for doing Encrypted Computation (
```
Comment (AlanAboudib) [inline README.md:14]:
```suggestion
SyferText is a library for privacy preserving Natural Language Processing in Python. It leverages PySyft to perform Federated Learning and Encrypted Computations (
```

File: README.md [hunk 3]
```diff
@@ -3,6 +3,22 @@
...
+[Multi-Party Computation (MPC)](https://en.wikipedia.org/wiki/Secure_multi-party_computation) and model training. The two main use cases of SyferText are:
```
Comment (AlanAboudib) [inline README.md:15]:
```suggestion
[Multi-Party Computation (MPC)](https://en.wikipedia.org/wiki/Secure_multi-party_computation) on text data. The two main usage scenarios of SyferText are:
```

File: README.md [hunk 4]
```diff
@@ -3,6 +3,22 @@
...
+
+- :fire: Secure plaintext pre-processing: Pre-process text data located on a remote server without breaking the privacy of data.
+- :rocket: Secure Pipeline Deploy: Encapsulate the complete pipeline of pre-processing and model securely and deploy it.
+  
+To get more detailed introduction on SyferText watch :movie_camera: [OpenMined AMA](https://youtu.be/gtwOPxxJ5F4) with Alan Aboudib on SyferText.
```
Comment (AlanAboudib) [inline README.md:20]:
```suggestion
- :fire: **Secure plaintext pre-processing:** Enables pre-processing of  text located on a remote machine without breaking data privacy.
- :rocket: **Secure pipeline deploy:** Starting from version 0.1.0, SyferText will be able to bundle the complete pipeline of pre-processing components and trained PySyft models and to securely deploy it to PyGrid.
  
To get a more detailed introduction about SyferText, watch :movie_camera: [OpenMined AMA Session](https://youtu.be/gtwOPxxJ5F4) with Alan Aboudib available on YouTube.
```

**Pull request merged**